ggshield/verticals/sca/file_selection.py

```diff
@@ -16,9 +16,11 @@
 
 # List of filepaths to ignore for SCA scans
 SCA_EXCLUSION_REGEXES = {
-    re.compile(r".*/" + re.escape(pattern) + "/.*")
+    re.compile(re.escape(pattern) + "/.*")
     for pattern in (
         "__pycache__",
+        "front",
+        "docs",
```

Comment:
I don't get why you add `front` and `docs` ? 
In some cases, those will be useful and it should be the user's choice to ignore those.